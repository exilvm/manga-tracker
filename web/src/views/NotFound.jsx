import React from 'react';
import { Paper, Typography } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  paper: {
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: theme.palette.background.paper,
    minHeight: '100vh',
    verticalAlign: 'center',
  },
  text: {
    textAlign: 'center',
    color: theme.palette.text.primary,
    alignItems: 'center',
  },
}));

export default function NotFound() {
  const classes = useStyles();

  return (
    <Paper className={classes.paper} square>
      <Typography className={classes.text} component='h1' variant='h1'>
        404 Not found
      </Typography>
    </Paper>
  );
}
