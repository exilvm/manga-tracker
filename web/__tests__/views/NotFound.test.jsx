import React from 'react';
import { create } from 'react-test-renderer';
import NotFound from '../../src/views/NotFound';

describe('404 page should render correctly', () => {
  it('Should match snapshot', () => {
    expect(create(<NotFound />)).toMatchSnapshot();
  });
});
