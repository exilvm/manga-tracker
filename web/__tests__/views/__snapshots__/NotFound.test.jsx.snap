// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page should render correctly Should match snapshot 1`] = `
<div
  className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation1"
>
  <h1
    className="MuiTypography-root makeStyles-text-2 MuiTypography-h1"
  >
    404 Not found
  </h1>
</div>
`;
